src/main/java/edu/kis/powp/jobs2d/drivers/adapter/LineDrawerAdapter.java

```diff
@@ -4,16 +4,12 @@
 import edu.kis.legacy.drawer.shape.ILine;
 import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.jobs2d.Job2dDriver;
+import static edu.kis.powp.jobs2d.features.DrawerFeature.getDrawerController;
 
-/**
- * driver adapter to drawer with several bugs.
- */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+
+public class LineDrawerAdapter extends DrawPanelController implements Job2dDriver {
```

Comment:
What about your naming proposal?
The name is better but it lacks a driver aspect.
Inheritance is not used (no super drawLine calls). Should be removed!